src/visualization/types/Band/options.tsx

```diff
@@ -174,7 +174,9 @@ const BandViewOptions: FC<Props> = ({properties, results, update}) => {
           widthMD={Columns.Six}
           widthLG={Columns.Four}
         >
-          <h5 className="view-options--header">Data</h5>
+          <h5 className="view-options--header" style={{marginTop: 0}}>
```

Comment:
this element has a classname, let's apply the style to that classname, rather than as an inline style. This component will be rendered over a graph, so we should do everything we can to minimize performance issues like inline style re-renders. This is doubly true since this style looks like it's being applied to every `h5` optiosn header